Xamarin.Forms.Platform.GTK/GtkDispatcher.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Xamarin.Forms.Platform.GTK
+{
+	public class GtkDispatcher : IDispatcher
```

Comment:
Is there any reason in the name of the class `GtkDispatcher`, instead of the `Dispatcher` as in other platforms?

Reply:
I followed the naming pattern of the other classes.